doc/readme.md

```diff
@@ -0,0 +1,16 @@
+# Trapperkeeper Authorization
```

Comment:
I'd be inclined to not include this new file and instead work any new content you think would be useful about extensions into the existing top-level `README.md` file in the repo.  Sound reasonable?